<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QOrbitalWidgetBase</class>
 <widget class="QWidget" name="QOrbitalWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="_labelIsovalue">
       <property name="text">
        <string>Isovalue</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSlider" name="_sliderIsovalue" native="true"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="_labelThreshold">
       <property name="text">
        <string>Threshold AO Selector</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="_thresholdSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>0.000010000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000050000000000</double>
       </property>
       <property name="value">
        <double>0.001000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="_labelGridResolution">
       <property name="text">
        <string>Grid Resolution</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="_sliderGridResolution">
       <property name="maximum">
        <number>8</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="_labelTransparence">
       <property name="text">
        <string>Transparence</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="_sliderTransparence">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>5</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="_labelOrbitals">
         <property name="text">
          <string>Molecular Orbitals</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="_listOrbitals">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="_labelOrbitals2">
         <property name="text">
          <string>Beta Molecular Orbitals</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="_listOrbitals2">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="_buttonHomo">
       <property name="text">
        <string>HOMO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_buttonLumo">
       <property name="text">
        <string>LUMO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_buttonTotalDensity">
       <property name="text">
        <string>TOTAL DENSITY</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_buttonSpinDensity">
       <property name="text">
        <string>SPIN DENSITY</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_buttonShowContours">
       <property name="text">
        <string>SHOW CONTOURS</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="_contoursGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QFrame" name="_frameSpectrogram">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="_buttonContour">
          <property name="text">
           <string>Contour</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="_buttonSpectrogram">
          <property name="text">
           <string>Spectrogram</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="_buttonPrint">
          <property name="text">
           <string>Print</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="_checkBoxXY">
            <property name="text">
             <string>XY</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="_spinBoxXY"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QCheckBox" name="_checkBoxXZ">
            <property name="text">
             <string>XZ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="_spinBoxXZ"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QCheckBox" name="_checkBoxYZ">
            <property name="text">
             <string>YZ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="_spinBoxYZ"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QDoubleSlider</class>
   <extends>QWidget</extends>
   <header>qdoubleslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
